karapace/schema_reader.py

```diff
@@ -38,6 +43,17 @@ def parse_jsonschema_definition(schema_definition: str) -> Draft7Validator:
     return Draft7Validator(schema)
 
 
+def parse_protobuf_schema_definition(schema_definition: str) -> ProtobufSchema:
```

Comment:
This function can be removed and `ProtobufSchema` can be called directly

Reply:
Yes but I prefer to keep it in this view to have the similar code looks.